README.md

```diff
@@ -222,7 +214,43 @@ This command does not touch repositories in which there are uncommitted changes.
 Examples:
 
 ```bash
-mgit update --recursive
+mgit update -- --recursive
+```
+
+### pull
+
+Pulls changes in existing repositories.
+
+If any dependency is missing, the command will install it too.
```

Comment:
At what branch?

Reply:
`pull` won't clone missing repositories anymore. It will work only for `update` command.